lib/active_admin/filters/active_filter.rb

```diff
@@ -66,7 +66,8 @@ def find_class
         if condition_attribute.klass != resource_class && condition_attribute.klass.primary_key == name.to_s
           condition_attribute.klass
         else
-          assoc = condition_attribute.klass.reflect_on_all_associations.detect { |r| r.foreign_key.to_s == name.to_s }
+          assoc = condition_attribute.klass.reflect_on_all_associations.
+            detect { |r| r.foreign_key.to_s == name.to_s && !r.options[:polymorphic] } #skip polymorphic
```

Comment:
Since we have the comment here I think we could improve it by making the code more explicit/readable. Would something like the below work? What do you think? Not critical so I leave it up to you.

```ruby
assoc = condition_attribute.klass.reflect_on_all_associations.
  reject { |r| r.options[:polymorphic] }.
  detect { |r| r.foreign_key.to_s == name.to_s }
```